### DISCUSSION TITLE
Wert (Integer) auf KNX-Gruppenadresse senden

### DISCUSSION THREAD

**Discussion by ghost** [jensweigele/ioBroker.yahka#243] [Q&A]
Hallo zusammen,

Leider finde ich keine Möglichkeit ein Device anzulegen, welches bei Betätigung einen festen Wert (Integer) auf eine KNX-Gruppenadresse senden kann. Bei einer solchen Szenennebenstelle gibt es keinen Wert den Homekit lesen könnte.

Kann jemand weiterhelfen? Besten Dank im Voraus.

Viele Grüße
binaryaddict

**Comment by unstressable** [comment]
Du könntest einen Schalter (Switch) anlegen, der dann ein Blockly auslöst, welches den Wert an KNX sendet. Am Ende des Blocklys kannst du den Schalter direkt wieder deaktivieren, dann hast du eine Art Taster.
Taster direkt hat Homekit nämlich nicht.

**Comment by Videonisse** [comment]
I have solved it with Lightbulb and using the value for Brightness as the Scene number (the KNX scene number is for me -1 in ioBroker, for example, should it be 60 in KNX it is 59 in ioBroker). 

In the Home app, I have created a Scenario for each scene number I use and then it will activate the Brightness with xx to control the KNX scene. 

![image](https://user-images.githubusercontent.com/13684218/134785880-10d06932-89c0-49ee-8396-7eeda49879ae.png)
